src/Lykke.Service.Zcash.Api.AzureRepositories/Operations/OperationRepository.cs

```diff
@@ -82,7 +83,7 @@ public OperationRepository(IReloadingManager<string> connectionStringManager, IL
 
         public async Task<IOperation> UpdateAsync(Guid operationId,
             DateTime? sentUtc = null, DateTime? minedUtc = null, DateTime? completedUtc = null, DateTime? failedUtc = null, DateTime? deletedUtc = null,
-            string hash = null, string error = null, uint? block = null)
+            string hash = null, string error = null, BlockchainException.ErrorCode? errorCode = null, uint? block = null)
```

Comment:
Nullable value of enum type. Won't it be more clear to add a special enum constant for "no error" case?

Reply:
I think no. This is generic approach here - null means we do not want to update value.